src/utils/logs.py

```diff
@@ -15,7 +15,8 @@ def __init__(self, config, exp_type):
         self.curr_iters = 0
         self.total_epochs = config["total_epochs"]
         self.items = {}
-        self.metrics = MetricLoader()
+        self.metrics = MetricLoader() 
+        # self.metrics = MetricLoader(device = "cpu") # need to pass device as CPU to not throw errors in non-CUDA machines
```

Comment:
Metricloader should figure out on its own whether the device is cpu

Reply:
Cool! I'll fix that!